templates/secret.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: sc-secret
+  annotations:
+    tekton.dev/docker-0: https://docker.io
+type: kubernetes.io/basic-auth
+stringData:
+  src_username: <>
+  src_password: <>
+  dest_username: <>
+  dest_password: <>
```

Comment:
Instead of having this Secret part of the templates, we should have a `kubectl` example. For instance [here](https://github.com/otaviof/task-git/blob/main/README.md#basic-auth-httpssh), however the Secret type is `--type=kubernetes.io/dockerconfigjson`.